src/api/features.api.js

```diff
@@ -572,7 +572,7 @@ export const identify = (
  *   be expressed
  * @returns {Promise<LayerFeature>}
  */
-const getFeature = (layer, featureID, lang = 'en', projection = WEBMERCATOR) => {
+const getFeature = (layer, featureID, lang = 'en', projection) => {
```

Comment:
It would be better to swap `lang` and `projection` if we remove the default value, it is unadvised to have non-default parameters declared after one that have default value

Reply:
you are right I'll change that